tests/outputs.test.ts

```diff
@@ -0,0 +1,34 @@
+import * as Nexus from 'nexus'
+import { generateSchemaAndTypesWithoutThrowing } from './__utils'
+
+it('only publishes output types that do not map to prisma models', async () => {
+  const datamodel = `
+  model User {
+    id  Int @id
+    name String
+  }
+`
+  const Query = Nexus.objectType({
+    name: 'Query',
+    definition(t: any) {
+      t.crud.user()
+    },
+  })
+
+  const Mutation = Nexus.objectType({
+    name: 'Mutation',
+    definition(t: any) {
+      t.crud.updateManyUser({ filtering: true })
+    },
+  })
+
+  const {
+    schema,
+    typegen,
+    missingTypes,
+  } = await generateSchemaAndTypesWithoutThrowing(datamodel, [Query, Mutation])
+
+  expect(Object.keys(missingTypes)).toContain('User')
+  expect(schema).toMatchSnapshot('schema')
+  expect(typegen).toMatchSnapshot('typegen')
```

Comment:
I wonder if we need snapshots for these non-integration tests.

Reply:
Removed the snapshots, you're right. I added the snapshot because I wanted to make sure that the missing type was properly replaced in the schema, but it's out of scope for this test